systemd/systemd.go

```diff
@@ -118,6 +121,17 @@ var systemctlCmd = func(args ...string) ([]byte, error) {
 	return bs, nil
 }
 
+func MockSystemdVersion(version int, injectedError error) (restore func()) {
```

Comment:
The injectedError is not used yet AFAICT but AIUI this will be used in a followup PR(?). Maybe a trivial test that ithe mocking works as expected? in systemd_test.go or something?

Reply:
No, I don't have a plan to use it, but I might :-) It was actually Pavel who suggested to add the error, and I think it can help to test some corner cases.

I've now added a unit test for it, so it won't appear to be unused anymore :-)